lib/recaptcha/configuration.rb

```diff
@@ -28,14 +28,18 @@ module Recaptcha
   #   end
   #
   class Configuration
-    attr_accessor :skip_verify_env, :secret_key, :site_key, :proxy, :handle_timeouts_gracefully, :hostname
+    attr_accessor :skip_verify_env, :secret_key, :site_key, :api_server_url, :verify_url, :proxy,
+      :handle_timeouts_gracefully, :hostname
 
     def initialize #:nodoc:
       @skip_verify_env            = %w[test cucumber]
       @handle_timeouts_gracefully = HANDLE_TIMEOUTS_GRACEFULLY
 
       @secret_key           = ENV['RECAPTCHA_SECRET_KEY']
       @site_key             = ENV['RECAPTCHA_SITE_KEY']
+
+      @api_server_url       = ENV['RECAPTCHA_API_SERVER_URL']
```

Comment:
Sorry, I don't quite understand. Actually I tried to follow how `@secret_key` and `@site_key` are set, except that if client never configures the url (or set via `ENV`), it will fallback to the CONFIG. Can you please explain further on how I can improve this?